Libplanet/Store/Trie/MerkleTrie.cs

```diff
@@ -60,6 +60,11 @@ public MerkleTrie(IKeyValueStore keyValueStore, INode? root = null, bool secure
         /// <inheritdoc/>
         public void Set(byte[] key, IValue value)
         {
+            if (value is null)
+            {
+                return;
```

Comment:
Rather than ignoring `null`, it would be better to throw an [`ArgumentNullException`][1] instead.  (You would need to update the XML comment in the interface file as well…)

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.argumentnullexception?view=netstandard-2.0